yellowbrick/classifier/confusion_matrix.py

```diff
@@ -77,9 +79,10 @@ def __init__(self, model, ax=None, classes=None, **kwargs):
         self.confusion_matrix = None
 
         self.cmap = color_sequence(kwargs.pop('cmap', 'YlOrRd'))
-        self.cmap.set_under(color='w')
-        self.cmap.set_over(color='#2a7d4f')
-        self.edgecolors=[] #used to draw diagonal line for predicted class = true class
+        self.cmap.set_under(color = 'w')
+        self.cmap.set_over(color = '#2a7d4f')
```

Comment:
Minor, and I'm not sure when this got added, but since I'm looking at it -- the hard coded color hex values make me nervous, they're difficult to find and debug. Would you mind making this a module variable, e.g. 

```python
CMAP_OVERCOLOR = '#2a7d4f' 

class ConfusionMatrix(Visualizer):
    ... 

    def __init__(self):
        self.cmap.set_over(color = CMAP_OVERCOLOR) 
```

I'd really appreciate it !